changelog/18.0/18.0.0/release_notes.md

```diff
@@ -0,0 +1,303 @@
+# Release of Vitess v18.0.0
+## Summary
+
+### Table of Contents
+
+- **[Major Changes](#major-changes)**
+  - **[Breaking Changes](#breaking-changes)**
+    - [Local examples now use etcd v3 storage and API](#local-examples-etcd-v3)
+  - **[New command line flags and behavior](#new-flag)**
+    - [VTOrc flag `--allow-emergency-reparent`](#new-flag-toggle-ers)
+    - [VTOrc flag `--change-tablets-with-errant-gtid-to-drained`](#new-flag-errant-gtid-convert)
+    - [ERS sub flag `--wait-for-all-tablets`](#new-ers-subflag)
+    - [VTGate flag `--grpc-send-session-in-streaming`](#new-vtgate-streaming-sesion)
+  - **[Experimental Foreign Key Support](#foreign-keys)**
+  - **[VTAdmin](#vtadmin)**
+    - [Updated to node v18.16.0](#update-node)
+  - **[Deprecations and Deletions](#deprecations-and-deletions)**
+    - [Deprecated Flags](#deprecated-flags)
+    - [Deleted Flags](#deleted-flags)
+    - [Deprecated Stats](#deprecated-stats)
+    - [Deleted `V3` planner](#deleted-v3)
+    - [Deleted `k8stopo`](#deleted-k8stopo)
+    - [Deleted `vtgr`](#deleted-vtgr)
+    - [Deleted `query_analyzer`](#deleted-query_analyzer)
+    - [Deprecated VTBackup stat `DurationByPhase`](#deprecated-vtbackup-stat-duration-by-phase)
+    - [Deprecated VDiff v1](#deprecated-vdiff-v1)
```

Comment:
I feel like we should re-order this to have either all `deprecated` items first or all `deleted` items first